src/App.tsx

```diff
@@ -1,109 +1,80 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const TOTAL_ITEMS = 42;
+const DEFAULT_PER_PAGE = 5;
+const PER_PAGE_OPTIONS = [3, 5, 10, 20];
+
+const items = getNumbers(1, TOTAL_ITEMS).map(n => `Item ${n}`);
```

Comment:
Ensure that the `getNumbers` function from `./utils` returns an array of numbers as expected. This is crucial for generating the list of items correctly.